npms/skale-ima/imaSgxExternalSigner.mjs

```diff
@@ -6,7 +6,7 @@ import * as rpcCall from "../../agent/rpcCall.mjs";
 
 const gIsDebugLogging = false; // development option only, must be always false
 const isColors = owaspUtils.toBoolean( process.argv[2] );
-cc.enable( true );
+cc.enable( false );
```

Comment:
What is the purpose of this?

Reply:
This is another external command, where I turned off colorization by default.
Nevertheless it prints only critical errors, it returns error text as is to caller process also.
Just keeping all colors off everywhere.